### DISCUSSION TITLE
v2/zh/guide/layout/home

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#305] [Announcements]
# v2/zh/guide/layout/home

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/layout/home.html

**Comment by yue1209** [comment]
23

**Comment by hellobambi** [comment]
如何自定义主页？比如修改主页的布局，添加背景等？

**Reply by licheng1013** [reply]
这些可能都要自己改。

**Comment by wywl99** [comment]
REDME.md 设置 home: false 后部署到 gitee 会显示成一篇文章且有两个特别大的图标
